src/app/tracing/dialog/dialog-movable/dialog-movable.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="fcl-dialog-movable">
+    <div
+        mat-dialog-content
+        cdkDrag
+        cdkDragRootElement=".cdk-overlay-pane"
+        cdkDragBoundary=".cdk-overlay-container"
+    >
+        <div class="fcl-drag-handle">
+            <mat-icon cdkDragHandle fill="currentColor">open_with</mat-icon>
+        </div>
+        <ng-template [ngIf]="template === 'dataImportWarning'">
+            <fcl-data-import-warning-modal></fcl-data-import-warning-modal>
+        </ng-template>
```

Comment:
This should utilize https://v16.angular.io/api/core/ng-content 

This way the component doesn't need to know about the content it will be rendering.

Reply:
- [ ] use ng-content